nerdlets/create-flow/blank.js

```diff
@@ -1,46 +1,28 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
 
-import { Button, HeadingText, TextField } from 'nr1';
+import { Button, HeadingText, InlineMessage, TextField } from 'nr1';
 
 import { Select } from '../../src/components';
 import { REFRESH_INTERVALS } from '../../src/constants';
 
 const BlankFlow = ({ accountId, accounts = [], onCreate, onCancel }) => {
   const [flowName, setFlowName] = useState('');
   const [selectedAccountId, setSelectedAccountId] = useState();
+  const [selectedAccountName, setSelectedAccountName] = useState();
   const [selectedRefreshInterval, setSelectedRefreshInterval] = useState(
     REFRESH_INTERVALS[0]
   );
 
   useEffect(() => {
-    if (!selectedAccountId) setSelectedAccountId(accountId);
+    if (!selectedAccountId) {
+      let accountName = (accounts.find((acc) => acc.id === accountId) || {})
+        .name;
+      setSelectedAccountName(accountName);
+      setSelectedAccountId(accountId);
+    }
```

Comment:
Looks like you're using this in both `blank.js` and `import.js`. I'd move this to the index file, and just send account name down to these components.